packages/box/lib/utils/unbox.ts

```diff
@@ -9,7 +9,14 @@ import inquirer from "inquirer";
 import { boxConfig, unboxOptions } from "typings";
 import { promisify } from "util";
 
-async function verifyURL(url: string) {
+function verifyLocalPath(url: string) {
+  const configPath = path.join(url, "truffle-box.json");
+  fse.access(configPath).catch(e => {
+    throw new Error(`Truffle Box at path ${url} doesn't exist.`);
+  });
+}
+
+async function verifyGithubURL(url: string) {
```

Comment:
I don't think we should make this particular to GitHub as I think urls for Bitbucket will work as well.

Reply:
yeah, but the current implementation (I didn't touch it at all) works only for github, that's why I named it like that. It could be made to work with any repo (actually `download-git-repo` has code to verify such URL already), but it is out of scope for this PR

Reply:
I think the current implementation will actually work for Bitbucket. `normalizeUrl` will just pass a full url through [here](https://github.com/trufflesuite/truffle/blob/develop/packages/box/box.ts#L34-L37). Then that url gets passed directly to `download-git-repo` [here](https://github.com/trufflesuite/truffle/blob/develop/packages/box/lib/utils/unbox.ts#L40-L42). So it should support full Bitbucket urls as well. Maybe I missed something?

Reply:
Yeah, you are right.. I got confused by the [line 24](https://github.com/trufflesuite/truffle/pull/3281/files#diff-a853c7f30d7bb0a9d440cdc429f87ee6R24) and thus thought it is only for Github